tests/PHPStan/Analyser/data/bug-8803.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Bug8803;
+
+use function PHPStan\Testing\assertType;
+
+class HelloWorld
```

Comment:
I think we should add a test fot constant numeric-strings which are not integer like

Reply:
Test added